### DISCUSSION TITLE
What can I clean up in a fresh install?

### DISCUSSION THREAD

**Discussion by benlilley** [statamic/cms#5339] [Q&A] [answered]
This is probably a bit of a strange question, but I'm starting a new project, and I always like having my root folder as clean and uncluttered as possible. What I'm struggling to work out (even though this is my third Statamic project) is what can I delete / what do I need to keep? For example:

- I'll be using gulp to build/compile, do I need to keep the `webpack.mix.js`?
- Can I remove files like `.styleci.yml` and `phpunit.xml`?
- Do I need to keep the default package.json file or can I dump it and start with my own?
- What folders can I remove like `tests/` and add back in later if I need to (say I want to build my own module later)?

In short, I want to delete anything Statamic doesn't need to run, and have the most barebones folder/file structure as I can.

**Comment by jasonvarga** [comment, marked as answer]
You can delete:

- database/
- tests/
- .styleci.yml
- README.md
- package.json
- phpunit.xml
- server.php
- tailwind.config.js
- webpack.mix.js

**Reply by benlilley** [reply]
Thanks @jasonvarga appreciate that!
